ecs/examples/simple/main.tf

```
@@ -56,4 +60,26 @@ resource "aws_security_group_rule" "simple" {
   protocol          = "tcp"
   security_group_id = aws_security_group.simple.id
   cidr_blocks       = ["0.0.0.0/0"]
-}
\ No newline at end of file
+}
+
+
+####################################################################################
+# IAM Roles
+####################################################################################
+
+data "aws_iam_policy_document" "simple" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+
+  }
+}
+
+resource "aws_iam_role" "simple" {
+  name               = "simple"
+  assume_role_policy = data.aws_iam_policy_document.simple.json
+}
```

Could we also attach the ECS task role policy so the example works:
```suggestion
}

resource "aws_iam_role_policy_attachment" "simple" {
  role       = aws_iam_role.simple.name
  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
}
```